pages/content/amp-dev/documentation/guides-and-tutorials/develop/interactivity_guide/ready_made.md

```diff
@@ -0,0 +1,147 @@
+---
+formats:
+  - websites
+$title: Ready-made interactivity with AMP components
+$order: 2
+description: "Read this guide to learn how to build interactivity in AMP."
+author: CrystalOnScript
+contributors:
+  - sbenz
+---
+
+This guide outlines interactive possibilities available in the  [AMP component library](../../../components/index.html). The library provides ready-made elements that meet common and uncommon user interface needs. You are able to customize these components, and how they interact with users, to meet specific needs. Read-on to learn how to harness this to decrease initial build costs without losing adaptability.
+
+# Ready-made interactive components
+
+Many of the available components fulfill a single purpose of interactivity. Most can standalone, but a core principle of AMP is composability. Composability allows you to combine different AMP components together to implement a customized interaction. Most allow templating with [amp-mustache](https://amp.dev/documentation/components/amp-mustache/).
+
+AMP offers the following ready-made interactive components:
```

Comment:
Suggestion: We could also include `amp-app-banner`, `amp-image-lightbox`, `amp-image-slider`, `amp-truncate-text`, `amp-video`, `amp-video-docking`

Reply:
Fantastic, thanks for flagging these!